front/pages/api/w/[wId]/labs/transcripts/index.ts

```diff
@@ -118,6 +119,7 @@ async function handler(
           workspaceId: owner.id,
           provider,
           connectionId,
+          defaultForWorkspace: req.body.defaultForWorkspace == true,
```

Comment:
You should check if there is not one already, relying in the DB constraint isn't a good practice.